### PR TITLE
[Builtins] Add another variant

### PR DESCRIPTION
**Pull request by effectfully**
Don't look here yet.

### CODE REVIEW

File: plutus-core/untyped-plutus-core/test/Evaluation/Builtins/SignatureVerification.hs [hunk 1]
```diff
@@ -75,8 +75,11 @@ ed25519Prop semvar = do
   cover 18 "happy path" . is (_Shouldn'tError . _AllGood) $ testCase
   runTestDataWith semvar testCase id VerifyEd25519Signature
 
+ed25519_Variant0Prop :: PropertyT IO ()
+ed25519_Variant0Prop = ed25519Prop (DefaultFunSemanticsVariant0)
```
Comment (kwxm) [inline plutus-core/untyped-plutus-core/test/Evaluation/Builtins/SignatureVerification.hs:79]:
```suggestion
ed25519_Variant0Prop = ed25519Prop DefaultFunSemanticsVariant0
```

File: plutus-core/untyped-plutus-core/test/Evaluation/Builtins/SignatureVerification.hs [hunk 2]
```diff
@@ -75,8 +75,11 @@ ed25519Prop semvar = do
...
+
 ed25519_Variant1Prop :: PropertyT IO ()
-ed25519_Variant1Prop = ed25519Prop DefaultFunSemanticsVariant1
+ed25519_Variant1Prop = ed25519Prop (DefaultFunSemanticsVariant1)
```
Comment (kwxm) [inline plutus-core/untyped-plutus-core/test/Evaluation/Builtins/SignatureVerification.hs:82]:
```suggestion
ed25519_Variant1Prop = ed25519Prop DefaultFunSemanticsVariant1
```

**Comment by kwxm** [comment]
> `-100.0%`

Impressive.

**Comment by effectfully** [comment]
Closing in favor of #5851.

**Pull request closed without merging**